inc/command/class-command.php

```diff
@@ -188,14 +202,224 @@ protected function phpunit( InputInterface $input, OutputInterface $output ) {
 		return $this->run_command( $input, $output, 'vendor/bin/phpunit', $options );
 	}
 
+	/**
+	 * Runs Codecept with zero config by default.
+	 *
+	 * @param InputInterface $input
+	 * @param OutputInterface $output
+	 * @return int
+	 */
+	protected function codecept( InputInterface $input, OutputInterface $output ) {
+		$options = $input->getArgument( 'options' );
+		$module = $input->getOption( 'module' );
+		$run_headless_browser = $input->getOption( 'browser' );
+		$use_chassis = $input->getOption( 'chassis' );
+		$tests_folder = $module !== 'project' ? "altis/$module/tests" : 'tests';
+		$project_subdomain = $this->get_project_subdomain();
+
+		// Write the default config.
+		$config = [
+			'paths' => [
+				'tests' => $tests_folder,
```

Comment:
Can this be dropped altogether in favour of the defaults or path specified by the command?

Reply:
Running `composer dev-tools codecept` without arguments gave me the following:

```
Running CodeCeption..

In Finder.php line 592:
                                                             
  The "/usr/src/app/vendor/tests" directory does not exist.  
                                                             

Removing test databases..
```

I think it's relative the config file so this should be `../tests` as the default tests directory I think.

Reply:
Specifying the path in the command makes sense, and will fix the wrong path issue.

Reply:
Also it seems that the `*.suite.yml` files have to exist in the tests directory for it pick them up, and you have to run `codecept build` before executing the tests. Just been trying with a chrome browser test.

We should perhaps use the contents of the `tests` directory you've added to this repo as a template to copy into the project's `tests` directory the first time this function is run, providing the files don't already exist.

I found they were required to run generator commands.

Reply:
I had thought this would be a manual thing, as engineers might want to mix and match modules and suites, create more or less. I mean this should serve as a template, but engineers would still need to figure out what they need, so automatic creation of the tests folder doesn't seem like the best idea.

Documentation should probably cover that. The `tests` folder in `dev-tools` can serve as a template for engineers to look at and/or copy from.

Reply:
I also fixed the default tests directory to be `../tests`, will update the command params to replace `-m` with `-p` which takes the complete path instead of a module name.

Reply:
I take your point. Along with documentation I'd like to have the option to generate the baseline suites and run `codecept build` in a follow up to this. e.g. `composer dev-tools scaffold codecept`, something along those lines.

Path option could be used to modify the `codeception.yml` output below to set the paths 👍